src/inference/smc.js

```diff
@@ -25,14 +25,17 @@ module.exports = function(env) {
       finalRejuv: true,
       saveTraces: false,
       importance: 'default',
-      onlyMAP: false
+      onlyMAP: false,
+      throwOnError: true
     });
 
     if (!_.includes(validImportanceOptVals, options.importance)) {
       var msg = options.importance + ' is not a valid importance option. ' +
           'Valid options are: ' + validImportanceOptVals;
       throw new Error(msg);
     }
+    this.throwOnError = options.throwOnError;
+    this.err = undefined;
```

Comment:
Am I right in thinking we don't need this property now we call `this.k` directly from `this.error`?